src/main/java/spark/route/RouteEntry.java

```diff
@@ -95,33 +97,28 @@ private boolean matchPath(String path) { // NOSONAR
         } else {
             // Number of "path parts" not the same
             // check wild card:
-            if (this.path.endsWith("*")) {
-                if (pathSize == (thisPathSize - 1) && (path.endsWith("/"))) {
-                    // Hack for making wildcards work with trailing slash
-                    pathList.add("");
-                    pathList.add("");
-                    pathSize += 2;
-                }
+            if (this.path.contains("*") && thisPathSize < pathSize) {
```

Comment:
NIT: '*' make it a constant?

Reply:
that's how it is in the spark code, so going to leave it as is.